Sources/MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -47,6 +62,50 @@ extension Bundle {
         return true
     }
     
+    /**
+     Returns a dictionary of `MBXInfo.plist` in Mapbox Core Navigation.
+     */
+    static let mapboxCoreNavigationInfoDictionary: [String: Any]? = {
+        guard let fileURL = Bundle.mapboxCoreNavigation.url(forResource: "MBXInfo", withExtension: "plist"),
+              let infoDictionary = NSDictionary(contentsOf: fileURL) as? [String: Any] else { return nil }
+        return infoDictionary
+    }()
+    
+    /**
+     Returns a dictionary of `MBXInfo.plist` in Mapbox Navigation framework bundle, if installed.
+     */
+    static let mapboxNavigationInfoDictionary: [String: Any]? = {
+        guard let fileURL = Bundle.mapboxNavigationIfInstalled?.url(forResource: "MBXInfo", withExtension: "plist"),
+              let infoDictionary = NSDictionary(contentsOf: fileURL) as? [String: Any] else { return nil }
+        return infoDictionary
+    }()
+    
+    /**
+     Returns the value associated with the specific key in the Mapbox Navigation bundle's  information property list, if installed.
+     */
+    public class func object(forMapboxNavigationInfoDictionaryKey key: String) -> String? {
+        if let stringForKey = Bundle.mapboxNavigationIfInstalled?.object(forInfoDictionaryKey: key) {
+            return stringForKey as? String
+        } else if let infoDictionary = Bundle.mapboxNavigationInfoDictionary {
+            return infoDictionary[key] as? String
```

Comment:
Unlike `Bundle.object(forInfoDictionaryKey:)`, this method is casting the return value as a string, preventing it from returning, say, a number. This is good enough for what we’re using the method for, but in that case we should rename the method to `string(forMapboxNavigationInfoDictionaryKey:)`.

Reply:
Thanks for the suggestion! I'll update the name of the method.